test/openAPI/features/data_exist.feature

```diff
@@ -1,19 +1,37 @@
-Feature: API endpoint that allows users to check if an entry exists in the Digital Registries database.
-  Request endpoint: POST /data/{registryname}/{versionnumber}/exists
+@method=GET @endpoint=/data/{registryname}/{versionnumber}/exists
+Feature: API endpoint that allows users to check if an entry exists in the database.
 
-  Scenario: The user receives a message that the record exists in the Digital Registries database
-    Given The user wants to check if a record exists in the Digital Registries database
-    When The user sends a valid request to check if a record exists in the database
-    Then The process of receiving a message that a record exists in the database completes successfully
-    And The user receives the information that a record exists in the database
+  @smoke
+    Scenario: Successfully receives a message that the record exists in database smoke type test
+    Given user wants to check if the searched record exists in the database
+    When send POST request to check if record exist in database with given path params "registryname" as registryname and "111" as versionnumber
+    And given body "ID" as ID and "FirstName" as FirstName and "LastName" as LastName and "BirthertificateID" as BirthCertificateID
+    Then response from /data/{registryname}/{versionnumber}/exist is received
+    And response should be returned in a timely manner 15000ms
+    And response should have status 200
+    And response should have content-type: application/json header
+    And response should match json schema
 
-  Scenario: The user receives the message that the record does not exist in the Digital Registries database
-    Given The user wants to check if the record exists in the Digital Registries database
-    When The user sends a valid request to check if a record exists in the database
-    Then The process of receiving a message that a record exists in the database completes successfully
-    And The user receives the information that the record does not exist in the database
+  @positive @unit
+    Scenario: Successfully receives a message that the record exists in database
```

Comment:
This test checks basically the same things as smoke test. I propose to add @unit and @positive tags to unit test.
Also add last And step to smoke test and remove the scenario